packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -95,6 +99,27 @@ class _GoogleMapsProviderState extends State<GoogleMapsProvider> {
   ArgumentCallback<LatLng> get onTap => widget.onTap;
   ArgumentCallback<LatLng> get onLongPress => widget.onLongPress;
   ArgumentCallback<AtlasController> get onMapCreated => widget.onMapCreated;
+  ArgumentCallback<CameraPosition> get onCameraPositionChanged =>
+      widget.onCameraPositionChanged;
+
+  PublishSubject<GoogleMaps.CameraPosition> publishSubjectCameraPosition;
+
+  @override
+  void initState() {
+    super.initState();
+
+    if (onCameraPositionChanged != null) {
+      publishSubjectCameraPosition =
+          PublishSubject<GoogleMaps.CameraPosition>();
+      _cameraPositionDebounceTime();
```

Comment:
I'm not sure about this. I think we should rename this function to something like `_setupCameraPositionListener`, and move this line into it: 

`publishSubjectCameraPosition = PublishSubject<GoogleMaps.CameraPosition>();`

Reply:
I totally agree! Thank you!